digits/inference/job.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
+from __future__ import absolute_import
+
+from . import tasks
+import digits.frameworks
+from digits.job import Job
+from digits.utils import override
+
+class InferenceJob(Job):
+    """
+    A Job that exercises the forward pass of a neural network
+    """
+
+    @override
```

Comment:
Thanks for the info I've been using those `@override` and `@subclass` without really understanding how they worked.